<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>319</width>
    <height>441</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>290</width>
     <height>410</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>290</width>
      <height>410</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(215, 193, 118
);
border-radius:10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditUsuario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
color:rgba(255,255,255);
padding-bottom:7px</string>
    </property>
    <property name="placeholderText">
     <string>Usuario</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditContra">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,0);
border:1px solid rgba(0,0,0,0);
border-bottom-color:rgba(46,82,101,255);
color:rgba(255,255,255);
padding-bottom:7px</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Contraseña</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BtnIngresar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>320</y>
      <width>250</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton{
background-color:rgba(70, 60, 29
);
color:rgba(255,255,255,200);
border-radius:5px}

QPushButton#pushButton:pressed{
padding-left:5px;
padding-right:5px;
background-color:rgba(127, 115, 77
);
baackground-position:calc(100%-10px)center;
}
QPushButton#pushButton:hover{
background-color:rgba(153, 144, 113
);
}
</string>
    </property>
    <property name="text">
     <string>Ingresar</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelimage">
    <property name="geometry">
     <rect>
      <x>66</x>
      <y>42</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
    <property name="pixmap">
     <pixmap>../../Imágenes/logopupa.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
